packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -132,28 +132,26 @@ void main() {
     semantics.dispose();
   });
 
-  testWidgets('Dialog default action styles', (WidgetTester tester) async {
+  testWidgets('Dialog default action style', (WidgetTester tester) async {
     await tester.pumpWidget(boilerplate(const CupertinoDialogAction(
       isDefaultAction: true,
       child: Text('Ok'),
     )));
 
     final DefaultTextStyle widget = tester.widget(find.byType(DefaultTextStyle));
 
-    expect(widget.style.fontWeight, equals(FontWeight.w400));
+    expect(widget.style.fontWeight, equals(FontWeight.bold));
   });
 
-  testWidgets('Default and destructive style', (WidgetTester tester) async {
```

Comment:
Wasn't this a legitimate test? If so, we should add a new test rather than rewrite an existing one.

Reply:
Due to the new assertion I can't set both parameters (`isDefaultAction`, `isDestructiveAction`) to true, the compiler warn: "Evaluation of this constant expression throws an exception. `dart(const_eval_throws_exception)`"

Reply:
Ok, makes sense. Thanks.

Reply:
`isDefaultAction` and `isDestructiveAction` seems to be orthogonal. On iOS an action can be both default(`preferredAction`) and destructive.